pkg/scheduler/factory.go

```diff
@@ -208,7 +188,7 @@ func (c *Configurator) createFromProvider(providerName string) (*Scheduler, erro
 		prof.Plugins = plugins
 	}
 
-	return c.create(extenders)
```

Comment:
no need to pass them, as they are part of `c` already.

Reply:
True, I had that to remain compatible with `createFromConfig` but I see that where it parses out the extenders can also just store them in `c`